modules/ps_emailsubscription/views/templates/hook/ps_emailsubscription.tpl

```diff
@@ -9,7 +9,7 @@
   <div class="container px-1">
     <div class="{$componentName}__content row">
       <div class="{$componentName}__content__left col-md-5">
-        <p class="{$componentName}__label">{l s='Get our latest news and special sales' d='Shop.Theme.Global'}</p>
+        <p id="block-newsletter-label" class="{$componentName}__label">{l s='Get our latest news and special sales' d='Shop.Theme.Global'}</p>
```

Comment:
```suggestion
        <p id="block-newsletter-label-{$hookName}" class="{$componentName}__label">{l s='Get our latest news and special sales' d='Shop.Theme.Global'}</p>
```

to avoid having two same `id`s if there are two newsletters on the website

(for example: in the footer, and in the popup)